projects/packages/videopress/src/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -202,5 +207,3 @@ public function videopress_block_update_meta( $request ) {
 		}
 	}
 }
-
-wpcom_rest_api_v2_load_plugin( 'WPCOM_REST_API_V2_Endpoint_VideoPress' );
```

Comment:
Why are we removing the call to `wpcom_rest_api_v2_load_plugin`? It seems to be gathering all endpoint instances in a big global array, don't we need this anymore?

Reply:
1 - because this function only exists in Jetpack. 2 - because I don't see us using this global anywhere for anything.. and we don't use that for other endpoints registered in packages...

Reply:
OK, it's good that you did the research - I was just kind of alarmed to see it being used in wpcom context as well, but if it's not needed - let's go with this change.

Reply:
I've confirmed that removing this line breaks VideoPress meta editing for Simple sites. What we need instead is:

```
if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
	wpcom_rest_api_v2_load_plugin( 'Automattic\Jetpack\VideoPress\WPCOM_REST_API_V2_Endpoint_VideoPress' );
}

```